server/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+)
+
+const (
+	exitFail = 1
+)
+
+func main() {
+	if err := run(os.Args); err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(exitFail)
+	}
+}
+
+func run(args []string) error {
+	port := flag.String("port", ":8080", "The server port with colon")
```

Comment:
Just being nitpicky here, but isn't a port with a colon actually an address? Just a shorter version of `*:8080`

Reply:
da hast du natuerlich recht.

Reply:
done @avorima